mentions/src/main/kotlin/io/spine/examples/pingh/mentions/GitHubClientProcess.kt

```diff
@@ -42,58 +45,103 @@ import io.spine.server.event.React
 import io.spine.server.procman.ProcessManager
 import kotlin.jvm.Throws
 
+/**
+ * Coordinates the updating of the specific user's mentions from GitHub.
+ */
 public class GitHubClientProcess :
     ProcessManager<GitHubClientId, GitHubClient, GitHubClient.Builder>() {
 
+    /**
+     * Service that fetches mentions from GitHub.
```

Comment:
Let's append the description with a link to `inject(..)`. We need to wire these things up in reader's head.